### PR TITLE
:seedling: Refactor the naming of transport configuration

### PR DESCRIPTION
**Pull request by yanmxa**
## Related issue(s)

Fixes # https://github.com/stolostron/multicluster-global-hub/pull/1116

## Tests
* [ ] Unit/function tests have been added and incorporated into `make unit-tests`.
* [ ] Integration tests have been added and incorporated into `make integration-test`.
* [ ] E2E tests have been added and incorporated into `make e2e-test-all`.
* [ ] List other manual tests you have done.

### CODE REVIEW

File: pkg/transport/kafka_credentail.go [hunk 1]
```diff
@@ -2,9 +2,9 @@ package transport
```
Comment (clyang82) [inline pkg/transport/kafka_credentail.go:2]: rename the filename to kafka_config.go

File: pkg/transport/rest_credentail.go [hunk 2]
```diff
@@ -2,7 +2,7 @@ package transport
```
Comment (clyang82) [inline pkg/transport/rest_credentail.go:2]: rename the filename to restful_config.go

File: pkg/transport/kafka_credentail.go (hunk 1 above)
Comment (yanmxa) [reply pkg/transport/kafka_credentail.go:2]: done

File: pkg/transport/rest_credentail.go (hunk 2 above)
Comment (yanmxa) [reply pkg/transport/rest_credentail.go:2]: done.

**Pull request merged**